pyflakes/test/test_undefined_names.py

```diff
@@ -366,15 +381,16 @@ def f2():
             global m
         ''', m.UndefinedName)
 
-    @skip("todo")
-    def test_unused_global(self):
-        """An unused global statement does not define the name."""
+    def test_unusedGlobalAcrossFunctions(self):
+        """
+        An unused "global" statement in another function does not define the name.
+        """
         self.flakes('''
         def f1():
-            m
+            global m
 
         def f2():
-            global m
+            m
         ''', m.UndefinedName)
```

Comment:
this test shouldn't have been altered -- it was demonstrating a behaviour that *should* happen

Reply:
at the moment, this is the one part of the diff i'm still iffy about. you're right, this should eventually just be un-skipped. `pyflakes` at the moment only errors when:

```python
def f1():
    global m

def f2():
    m
```

and it does not error if the global is defined in `f2`. not sure if this is some bigger thing with pyflakes or if it's a simple fix i can do in here!